server/src/pages/settings/controller.js

```diff
@@ -0,0 +1,32 @@
+/* browser: true */
+
+const page = require("./page").page;
+
+let model;
+
+exports.launch = function(m) {
+  model = m;
+  render();
+}
+
+exports.disconnectDevice = function() {
+  let url = model.backend + "/api/disconnect-device";
+  let req = new XMLHttpRequest();
+  req.open("POST", url);
+  req.setRequestHeader("content-type", "application/x-www-form-urlencoded");
+  req.onload = function() {
+    if (req.status >= 300) {
+      // FIXME: a lame way to do an error message
+      window.alert("Error disconnecting: " + req.status + " " + req.statusText);
+    } else {
+      location.href = model.backend + "/settings";
```

Comment:
We have a couple when the user clicks on disconnect, cancels or confirms, so we could skip this I think. @ianb